From a66b2d99f21bff6986b0dda5b7cc9ee393be934a Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Wed, 29 Aug 2018 12:30:14 +0300
Subject: [PATCH] build: Fix GSETTINGS_SCHEMA_DIR envvar handling in tests

GSETTINGS_SCHEMA_DIR does not support multiple colon-separated paths.
---
 src/tests/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/meson.build b/src/tests/meson.build
index 646f6f875..42c0e6d28 100644
--- a/src/tests/meson.build
+++ b/src/tests/meson.build
@@ -16,7 +16,7 @@ ide_test_env = [
   'G_TEST_BUILDDIR=@0@'.format(meson.current_build_dir()),
   'G_DEBUG=gc-friendly',
   'GSETTINGS_BACKEND=memory',
-  'GSETTINGS_SCHEMA_DIR=@0@/data/gsettings:$(GSETTINGS_SCHEMA_DIR)'.format(meson.build_root()),
+  'GSETTINGS_SCHEMA_DIR=@0@/data/gsettings'.format(meson.build_root()),
   'PYTHONDONTWRITEBYTECODE=yes',
   'MALLOC_CHECK_=2',
 #  'MALLOC_PERTURB_=$((${RANDOM:-256} % 256))',
-- 
2.17.0

